modules/Package.swift

```diff
@@ -60,9 +60,8 @@ let package = Package(
         .package(url: "https://github.com/pointfreeco/swift-composable-architecture", from: "0.59.0"),
         .package(url: "https://github.com/pointfreeco/swift-case-paths", from: "0.14.1"),
         .package(url: "https://github.com/pointfreeco/swift-url-routing", from: "0.5.0"),
-        .package(url: "https://github.com/SwiftGen/SwiftGenPlugin", from: "6.6.0"),
         .package(url: "https://github.com/zcash-hackworks/MnemonicSwift", from: "2.2.4"),
-        .package(url: "https://github.com/zcash/ZcashLightClientKit", from: "2.0.0-rc.4"),
+        .package(url: "https://github.com/zcash/ZcashLightClientKit", branch: "release-2.0.0"),
```

Comment:
I don't like specifying dependencies from branches for security reasons. If this is just temporary and will not get into a release, then ok.

Reply:
It was temporary during the development. Updated to the final release tag.